microscopium/screens/cellomics.py

```diff
@@ -29,8 +30,12 @@ def batch_stitch_stack(file_dict, output, order=[0, 1, 2], target_bit_depth=8, *
         files. This dictionary is built using the ``make_key2file`` function.
     output : string
         The directory to output the stitched and concatenated images to.
-    order : list of int, optional
+    stitch_order : array of int, shape (M, N)
+        The order of the stitching.
+        Passed to ``microscopium.cellomics.snail_stitch``
```

Comment:
I've got a bit more experience with sphinx docs now (check out gala.readthedocs.org!), turns out functions should be defined in-scope and in single ticks:

```
Passed to "order" argument of `snail_stitch`.
```

Then when sphinx renders the API docs it will have a hyperlink to that function!